### ISSUE TITLE
Pricing Import

### ISSUE THREAD

**Issue by tngweb** [gogetssl/whmcs-addon#185]
**Describe the solution you'd like**
I don't know if this is an issue or a request as there's nothing in the Wiki about this.

There should be a way to be able to sync the pricing from our account on GOGETSSL into WHMCS. I want to sell ALL products available, but it is tiring to manually enter all of the prices into the Product Configuration.

**Comment by vlad-evg** [comment]
Hello,

there is price AUTO Update in Product configuration
https://www.screencast.com/t/8HoW8WHXn9

**Comment by tngweb** [comment]
Ok then, there is an issue with this as it is not pulling the prices through the API. I follow the instructions to setup the addon, I am using the Multiple Product Creator to setup my products. In the Products Config., I set Auto Setup to after Payment, I set my commission, select Status Enabled, Select Price Auto Download, and select ALL products. and Save. The system does NOT import the prices. I have tried several times, even going through selecting the Price Enable for the 3 terms and still nothing. I even left it overnight and the CRON didn't update the prices.
